components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1478,17 +1429,32 @@ void RewardsServiceImpl::GetBalanceReport(
                      std::move(callback)));
 }
 
+void RewardsServiceImpl::GetPublisherActivityFromUrl(
+    mojom::VisitDataPtr visit_data) {
+  if (!Connected()) {
+    return;
+  }
+  uint32_t tab_id = visit_data->tab_id;
+  engine_->GetPublisherActivityFromUrl(tab_id, std::move(visit_data), "");
```

Comment:
I'm confused, below we're passing `windowId` to `GetPublisherActivityFromUrl`, but we're passing tabId here? Why would we ever be dealing with a window id?

Reply:
This is a legacy function - the `windowId` parameter name might event predate `brave-core`. I've updated the parameter name to be `tab_id` to reduce confusion.